src/utils/trimLine.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
-var urlOrContentRe = /(["'].+["'])|( +|:)url\(.+\)/
+var contentRe = /(content:\s*["'])(?:.+)(["'])/
+var urlRe = /([\S ]+\:\s*url\s*\(["'])(?:.+)(["']\))/
```

Comment:
This regexp matches anything before and after a url expression such as `background: url("../images/someimage.png") center center;` With that expression it would match `background: url("` and `") center center;`